src/routes/arrays.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+
+const arraysController = require('../controllers/arrays');
+
+const arraysRoute = express.Router();
+
+arraysRoute.post('/element-at-index/:number', arraysController.getElementNthIndex);
```

Comment:
These can be condensed down a little to group them appropriately. If you remove all the blank lines and run eslint on the file it will make it look like this:
```
const express = require('express');
const arraysController = require('../controllers/arrays');

const arraysRoute = express.Router();
arraysRoute.post('/element-at-index/:number', arraysController.getElementNthIndex);
```